charts/prometheus/README.md

```diff
@@ -59,7 +59,16 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 ### To 16.0
 
-Start from 16.0, prometheus moves all embedded charts out of prometheus chart umbrella back to its parent. Version 16.0.0 moves node-exporter to external promtheus-node-exporter. If you've made some manual changes, please diff the old (`nodeExporter`) and the new (`prometheus-node-exporter`) configuration section in values.yaml. Uninstalling the old chart release and then reinstalling the new chart release is recommended.
+Start from 16.0, prometheus moves all embedded charts out of prometheus chart umbrella back to its parent. Version 16.0.0 moves node-exporter to external promtheus-node-exporter. If you've made some manual changes, please diff the old (`nodeExporter`) and the new (`prometheus-node-exporter`) configuration section in values.yaml.
```

Comment:
Starting from version 16.0 embedded services (like alertmanager, node-exporter etc.) are moved out of Prometheus chart and the respecting charts from this repo are used as dependencies. Version 16.0.0 moves node-exporter service to [prometheus-node-exporter chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-node-exporter). If you've made some config changes, please check the old `nodeExporter` and the new `prometheus-node-exporter` configuration section in values.yaml for differences.